src/excited_configurations.jl

```diff
@@ -1,19 +1,60 @@
+"""
+    orbital_priority(fun, orig_cfg, orbitals)
+
+Generate priorities for the substitution `orbitals`, i.e. the
+preferred ordering of the orbitals in configurations excited from
+`orig_cfg`. `fun` can optionally transform the labels of substitution
+orbitals, in which case they will be ordered just after their parent
+orbital in the source configuration; otherwise they will be appended
+to the priority list.
+"""
+function orbital_priority(fun::Function, orig_cfg::Configuration{O₁}, orbitals::Vector{O₂}) where {O₁<:AbstractOrbital,O₂<:AbstractOrbital}
```

Comment:
```suggestion
function orbital_priority(f::Function, orig_cfg::Configuration{O₁}, orbitals::Vector{O₂}) where {O₁<:AbstractOrbital,O₂<:AbstractOrbital}
```

Feels more idiomatic to me, but just a suggestion.

Reply:
Then I have to change everywhere, so I'll leave it as-is for now.